compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/SwingPanel.desktop.kt

```diff
@@ -97,7 +97,7 @@ public fun <T : Component> SwingPanel(
         focusSwitcher.Content()
     }
 
-    DisposableEffect(factory) {
+    DisposableEffect(factory, update) {
```

Comment:
This is a change in Desktop sourceSet.
I think we need also add `update` function reference to the key because it stores in the Updater object.